linter/allowlists/de.txt

```diff
@@ -602,6 +602,8 @@ Vorschaubild
 Webanwendungen
 Webseite
 Webseitenwurzel
+Webseitenbesucher
```

Comment:
Dieses Wort scheint nicht im Beschreibungstext vorzukommen?

Reply:
It is the correct German word of "Websitebesucher". "Websitebesucher" was already in the file.

Reply:
Yes, but `Webseitenbesucher` is not used in your PR, so there is no need to add it. Please remove it again.